api/v1alpha1/openapi.yaml

```diff
@@ -2569,6 +2569,42 @@ components:
               - filePath
           required:
           - httpRef
+    ApplicationSpec:
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/ApplicationEnvVars'
+        - type: object
+          properties:
+            name:
+              type: string
+              description: "The name of the application"
+        - oneOf:
+            - $ref: '#/components/schemas/ImageApplicationProvider'
+            # extend application providers
+    ImageApplicationProvider:
+      type: object
+      properties:
+        image:
+          type: string
+          description: "Reference to the container image for the application package"
+    ApplicationEnvVars:
+      type: object
+      properties:
+        envVars:
```

Comment:
The name "envVars" suggests a specific mechanism how parameters are passed into the templating mechanism, which doesn't apply to Helm charts, for example. Not saying we need to change it, just wanted to point out this potential source of confusion.

Reply:
Will improve the docs.